app/extensions/brave/locales/en-US/newtab.properties

```diff
@@ -27,6 +27,8 @@ privateTabText1=Whether or not you use Tor, private tabs are not logged in page
 privateTabTitle=This is a Private Tab
 privateTabTorTitle=Make private tabs much more private with
```

Comment:
With this change, perhaps we should not make it seem like a global setting a more like a setting just for the current tab: 'Make this tab much more private with Tor'?